src/styles/book.css

```diff
@@ -0,0 +1,67 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;700&family=Roboto+Slab:wght@200;300;400;700&display=swap');
+
+.book-wrapper {
+  background-color: white;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  font-family: 'Roboto Slab', serif;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding: 2em;
+  height: 10rem;
+  margin-bottom: 2rem;
+}
+
```

Comment:
- Hey, great job with the styling, however, you're design is not yet close to the given template. please, kindly refer to the template to implement these missing parts of the design as much as you can to improve your design to meet the project requirement. see the images below:


*your design*
![Screenshot from 2022-11-12 01-18-00](https://user-images.githubusercontent.com/59397159/201446978-f84a8f95-b9ef-43ec-81a1-989488f4d4f9.png)

*design template*
![Screenshot from 2022-11-12 01-16-48](https://user-images.githubusercontent.com/59397159/201446980-611a0543-23e0-4c7e-9772-c1c58731d769.png)

- Please, kindly follow the design structure as seen in the images above :pray: